lint/lint.py

```diff
@@ -226,6 +226,14 @@ def check_parsed(repo_root, path, f, css_mode):
     if source_file.type == "visual" and not source_file.name_is_visual:
         return [("CONTENT-VISUAL", "Visual test whose filename doesn't end in '-visual'", path, None)]
 
+    for reftest_node in source_file.reftest_nodes:
+        href = reftest_node.attrib.get("href", "")
+        reference_file = os.path.join(repo_root, source_file.dir_path, href)
```

Comment:
That's true, but it begs the question: do we want to support absolute URLs? It seems fragile for reference tests to refer to external resources. Even root-relative URLs could be problematic--dependencies that span directories like that may not always be apparent, making it difficult to maintain one specification's files due to the ref tests in another specification.

It might be best to classify those patterns as linting errors as well. In that case, maybe a dedicated error message would be warranted. What do you think?